ivadomed/loader/loader.py

```diff
@@ -42,6 +42,7 @@ def load_dataset(data_list, bids_path, transforms_params, model_params, target_s
         dataset_type (str): Choice between "training", "validation" or "testing".
         requires_undo (bool): If True, the transformations without undo_transform will be discarded.
         object_detection_params (dict): Object dection parameters.
+        soft_gt (bool): If True, ground truth will be non-binarized images encoded in float32.
```

Comment:
why forcing float32 encoding? doesn't it work for e.g. uint8?

Reply:
With float32 we will have a mask with continuous value which is what we are looking for with the soft ground truth.

Reply:
ah, yes obviously uint8 would not work. I was thinking more of other floats, e.g. float64. I am just wondering why this specific float32 is expected, and if it is, why not asserting it in your code?

Reply:
We transform our tensor in float (to understand float32 following pytorch [naming convention](https://pytorch.org/docs/stable/tensors.html)) so greater precision does not seem relevant. I will add it to the documentation.

Reply:
so why not asserting if it is expected?

Reply:
Not sure if I understand. Uint8 is used to reduce the ram storage when we have binary mask.

Reply:
Oh right right! I will modify the documentation